css/src/atomics/border.scss

```diff
@@ -0,0 +1,55 @@
+.border {
+	border: $border-width solid $border !important;
+}
+
+.border-none {
+	border: 0 !important;
+}
+
+@include tablet {
+	.border-tablet {
+		border: $border-width solid $border !important;
+	}
+
+	.border-none-tablet {
```

Comment:
For flex atomics wherever the value was 0 we kept the value as a part of the CSS class so for example for `flex-shrink:0` we have css class as `.flex-shrink-0` and for tablet it should be `flex-shrink-0-tablet` , Dont you think it should be the same here for border value as 0 for uniformity ?

Reply:
So let me begin from the end: sass was screaming at me for the `border: none`, that's why I changed it to `border: 0` which is also a valid value that leads to the same visual result. But I don't want to mess around with naming, I think having `-none` is convenient and familiar to everyone :)

Reply:
I understand , personally i prefer `-none`  and i know its convenient and everyone knows about it but the real question here is what are we going to use in atlas wherever the value is `0` or `none` and i feel since we are in the beginning of the process we should come-up with some rule  ? I believe as a framework we should have a uniform naming convention . @wibjorn - what's your through on this ?

Reply:
I think it also depends on what's semantical would be correct. In your case with flex, there are just two variations for flex-shrink helpers' names - `flex-shrink-0` and `flex-shrink-1` and it matches exactly the value, which is kinda expected. It should be a number. But with border rules, we only adjust if there is a border or not. So it looks semantically correct to say `-none` in the class which removes the border. Same, for example, with `spacing` stuff (margin/padding rules) - it uses `-none` too and update it to `margin-0`, feels weird to me 🤔

Reply:
Currently, we have this stylelint rule `declaration-property-value-disallowed-list': { '/^border/': ['none'] }`. Therefore, is this a rule that we should keep or remove? I don't see anything inherently wrong with `border: none`

Reply:
@wibjorn please read and share your thoughts! 👆🏻

Reply:
Agreed that I have no qualms with `border: none`, feel free to remove it from the lint rules.

Reply:
the stylelint rule was removed, and the `border: 0` was replaced with `border: none`